DataFormats/PatCandidates/src/classes_def_objects.xml

```diff
@@ -149,7 +149,8 @@
    <version ClassVersion="10" checksum="2692173055"/>
   </class>
   <class name="std::vector<pat::tau::TauCaloSpecific>" />
-  <class name="pat::tau::TauPFEssential" ClassVersion="13">
+  <class name="pat::tau::TauPFEssential" ClassVersion="14">
```

Comment:
You can leave this stuff in but I would not add it to the next PR. We take this one on a separate note.